jboss/container/wildfly/s2i/bash/artifacts/opt/jboss/container/wildfly/s2i/galleon/s2i_galleon

```diff
@@ -15,6 +15,16 @@ function galleon_patch_generic_config() {
        sed -i "s|<!-- ##GALLEON_LAYERS## -->|${layers}|" $GALLEON_GENERIC_LAYERS_DEFINITION/config.xml
      fi
    fi
+   if [ -n "$GALLEON_PROVISION_FEATURE_PACKS" ]; then
+     echo Provisioning custom Galleon feature-packs: $GALLEON_PROVISION_FEATURE_PACKS
+     for fp in $(echo $GALLEON_PROVISION_FEATURE_PACKS | sed "s/,/ /g"); do
```

Comment:
I think it might be a lot safer to write this as -n "${GALLEON_PROVISION_FEATURE_PACKS}" especially if this could have spaces in the provided value.